### DISCUSSION TITLE
Live Measurement Issue

### DISCUSSION THREAD

**Discussion by TobTkyDev** [BeatsuDev/tibber.py#29] [Q&A] [answered]
Hello, thanks for building this wrapper. Q: I've used your example of live measurement but for me it doesn't work. I cannot seen any data in the shell. Do I need to call the callbacks somewhere? 

Thanks!

**Comment by ericbstie** [comment, marked as answer]
You don't need to call the callback functions, no. tibber.py will call them for you as long as you have registered them with home.event("live_measurement"). Could you create a GitHub issue on this instead and use the bug template? It'll be easier to reply and follow up 😄

**Reply by TobTkyDev** [reply]
Thanks! Just opened a ticket as you suggested.
